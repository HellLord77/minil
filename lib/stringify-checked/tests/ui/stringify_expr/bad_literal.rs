use stringify_checked::stringify_expr;

fn main() {
    stringify_expr!(tralse);
}
