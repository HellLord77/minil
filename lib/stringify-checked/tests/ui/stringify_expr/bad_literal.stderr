error[E0425]: cannot find value `tralse` in this scope
 --> tests/ui/stringify_expr/bad_literal.rs:4:21
  |
4 |     stringify_expr!(tralse);
  |                     ^^^^^^ not found in this scope

warning: use of deprecated macro `stringify_expr`
 --> tests/ui/stringify_expr/bad_literal.rs:4:5
  |
4 |     stringify_expr!(tralse);
  |     ^^^^^^^^^^^^^^
  |
  = note: `#[warn(deprecated)]` on by default

warning: use of deprecated macro `stringify_checked::stringify_expr`
 --> tests/ui/stringify_expr/bad_literal.rs:1:24
  |
1 | use stringify_checked::stringify_expr;
  |                        ^^^^^^^^^^^^^^
